cmd/ocm/create/cluster/cmd.go

```diff
@@ -46,6 +46,9 @@ const (
 	defaultIngressExcludedNamespacesFlag       = "default-ingress-excluded-namespaces"
 	defaultIngressWildcardPolicyFlag           = "default-ingress-wildcard-policy"
 	defaultIngressNamespaceOwnershipPolicyFlag = "default-ingress-namespace-ownership-policy"
+	gcpTermsAgreementsHyperlink                = "https://console.cloud.google.com" +
+		"/marketplace/agreements/redhat-marketplace/red-hat-openshift-dedicated"
+	gcpTermsAgreementError = "Please agree to Google Terms and Agreements in order to proceed"
```

Comment:
Same question here: should it be terms and conditions instead?

Reply:
Keeping agreements for consistency : https://docs.google.com/document/d/1xpRCZ0QYWRuvVtCEycWrHwXWSIepzUkgb8Nobqf3Dlc/edit#heading=h.ve1e69gtx5pl

Reply:
The console says "accept" instead of agree, should we update here too?

Reply:
Updated